### PR TITLE
Make links work when opened in a new tab using right click context menu

### PR DESCRIPTION
**Pull request by ibnesayeed**
This is an attempt to fix https://github.com/play-with-docker/play-with-docker.github.io/issues/126, but not tested well.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,6 +69,13 @@ import EventEmitter from 'wolfy87-eventemitter'
           window.open(protocol + '://'+ instance.proxy_host + '-' + port + '.direct.' + self.opts.baseUrl.split('/')[2] + this.attributes.href.value, '_blank');
         }
       };
+      actions[n].oncontextmenu = function(evt) {
```
Comment (marcosnils) [inline index.js]: Seems like this code is pretty similar to the one in `onclick`. Can we do something to avoid duplication?

**Comment by ibnesayeed** [comment]
Now, I have tested it and it seems to be working as expected.

**Comment by ibnesayeed** [comment]
@marcosnils I added the following line in the `index.html` file temporarily to test my changes:

```html
<a href="/" data-term="#myTerm" data-port="80">Click Here</a>
```

**Comment by marcosnils** [comment]
@ibnesayeed still doesn't work in my case. If you go to the `Docker for beginners` tutorial and try to left click the `click here` link in a new tab it still doesn't work :disappointed:

**Comment by ibnesayeed** [comment]
@marcosnils how did you try to test on the training site while these changes are not reflected there yet? The live site is using RawGit CDN from the `dist` directory. That means in order for it to be effective there, the change here needs to be merged and the dist files are to be regenerated. If you have any other means of testing it on the training site then I am not aware of it.

**Comment by ibnesayeed** [comment]
The RawGit CDN used by the PWD to serve the SDK files is sunsetting. https://github.com/play-with-docker/play-with-docker.github.io/issues/166

**Comment by ibnesayeed** [comment]
Serving `/dist/pwd.js` will not work until the dist is rebuilt because changes are made in the `index.js` file.

**Comment by marcosnils** [comment]
You can run `npm run build ` to generate the PWD.js built file.

sent from mobile

**Comment by ibnesayeed** [comment]
Yes, that's how I tested it locally earlier, but I thought you were missing that as you said it was not working for you while it does for me.

**Comment by marcosnils** [comment]
What it wasn't working for me has the middle button click, on the right
button. In any case it should work for all cases now

sent from mobile

**Comment by Deifinger** [comment]
It works well. Thank you guys @marcosnils @ibnesayeed

**Pull request merged**